src/store/index.ts

```diff
@@ -230,17 +234,29 @@ const useTaxesStore = defineStore({
       return this.taxableIncome - this.taxIncomeAvg;
     },
 
+    solidarityFeeTotal() {
+      if (this.taxableIncome < 80000) {
```

Comment:
Could you just add a comment here with the [link](https://info.portaldasfinancas.gov.pt/pt/informacao_fiscal/codigos_tributarios/irs/Pages/irs68a.aspx) you provided? Helpful for future reference!